common/components/internal-header/template.test.js

```diff
@@ -1,3 +1,4 @@
+const { AUTH_BASE_URL } = require('../../../config/index.js')
```

Comment:
I've left a comment on the template file as to how we can remove the need to include our app configuration from this component. I think it would be a much cleaner separation to keep the two separate.